modules/nanobind/2.2.0/MODULE.bazel

```diff
@@ -0,0 +1,9 @@
+module(
+    name = "nanobind",
+    version = "2.2.0",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "platforms", version = "0.0.10")
+bazel_dep(name = "robin-map", version = "1.3.0")
+bazel_dep(name = "rules_python", version = "0.36.0")
```

Comment:
1.0.0 is out - do we want to upgrade?

Reply:
the downside of doing that is it forces all consumers to use that at a minimum, so it's generally preferred that we use our lowest supported version here, so users can use `>=` this